taxonomy-editor-frontend/src/pages/editentry/ListTranslations.jsx

```diff
@@ -74,6 +68,17 @@ const ListTranslations = ({ nodeObject, setNodeObject }) => {
     setIsDialogOpen(false);
   };
 
+  const handleLanguagesChange = (newShownLanguageCodes) => {
+    const addedLanguageCodes = newShownLanguageCodes.filter(
+      (langCode) => !selectedShownLanguages.includes(langCode)
+    );
+    addedLanguageCodes.forEach((langCode) => {
+      handleAddTranslation(langCode);
```

Comment:
let's add a check in `handleAddTranslation` so it only adds a language if is not present. After doing that, we could just do `newShownLanguageCodes.forEach((langCode)=> {handleAddTranslation(langCode)})`

Reply:
I added the check in this function only, no need to call `handleAddTranslation` if a language is already present in the node